public/javascript/operatorConstants.js

```diff
@@ -0,0 +1,4 @@
+export const ADD = 'add'
+export const SUBTRACT = 'subtract'
+export const MULTIPLY = 'multiply'
+export const DIVIDE = 'divide'
```

Comment:
This definitely falls under the realm of over-engineering. These could easily just sit as strings in the code and it would be way more readable than having to jump between files